src/Integration/Service/SonarQubeServiceWrapper.cs

```diff
@@ -318,6 +326,11 @@ public Uri CreateProjectDashboardUrl(ConnectionInformation connectionInformation
 
         #region Helpers
 
+        internal /*for testing purposes*/ static string GetServerLanguageKey(Language language)
+        {
+            return languageKeys[language];
```

Comment:
This will throw for unrecognised languages. Would an assertion be enough to check that it's only called for one of the known languages, or does it need to be more robust?